app/views/AttachmentView.tsx

```diff
@@ -79,29 +98,17 @@ class AttachmentView extends React.Component {
 		}
 		const options = {
 			title,
-			headerLeft: () => (
-				<HeaderButton.CloseModal
-					testID='close-attachment-view'
-					navigation={navigation}
-					buttonStyle={{ color: themes[theme].previewTintColor }}
-				/>
-			),
+			headerLeft: () => <HeaderButton.CloseModal testID='close-attachment-view' navigation={navigation} />,
```

Comment:
why remove the buttonStyle ?

Reply:
There isn't buttonStyle when looking into the component